packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -183,28 +208,12 @@ export async function uploadCollectedData(context: SynchronizationJobContext) {
       walkDirectory({
         path: 'graph',
         async iteratee({ filePath }) {
-          const parsedData = await readGraphObjectFile<FlushedGraphObjectData>({
-            filePath,
-          });
-
-          try {
-            if (Array.isArray(parsedData.entities)) {
-              await uploadData(context, 'entities', parsedData.entities);
-            }
-
-            if (Array.isArray(parsedData.relationships)) {
-              await uploadData(
-                context,
-                'relationships',
-                parsedData.relationships,
-              );
-            }
-          } catch (err) {
-            throw synchronizationApiError(
-              err,
-              'Error uploading collected data',
-            );
-          }
+          await uploadGraphObjectData(
```

Comment:
I see that `uploadCollectedData` is still being called from `synchronizeCollectedData` (which is used in the `j1-integration sync` command). I haven't looked at the follow-on PRs yet so maybe you will do this, but I think it'd be unsafe to leave this functionality in the off chance that it duplicates uploads to the persister.

Reply:
I think we still want this. I don't personally use the `sync` command. `run` collects & uploads. `sync` only uploads. The `sync` command assumes that you already have data on disk that now should be uploaded.

Reply:
It seems kind of weird to me that there is any interaction with the persister at all in the open-source SDK as that is J1 specific. Am I wrong with the goal of the SDK?